modules/aws/ecs.go

```diff
@@ -36,7 +36,7 @@ func GetEcsClusterE(t *testing.T, region string, name string) (*ecs.Cluster, err
 
 	numClusters := len(output.Clusters)
 	if numClusters != 1 {
-		return nil, fmt.Errorf("Expected to find 1 ECS cluster named '%s' in region '%v', but found '%d'",
+		return nil, fmt.Errorf("expected to find 1 ECS cluster named '%s' in region '%v', but found '%d'",
```

Comment:
We like to keep error messages as sentences. Can you keep the capitalization here and also in all the new functions below?

Reply:
I knew this would trigger a discussion! Intellij idea reported as a linting error so I fixed that. 
OK, will do. Following a convention makes much more sense.